Models/topics.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../db/connection.js');
+
+function fetchTopics() {
+    const query = 'SELECT * FROM topics;';
+    return db
+    .query(query)
+    .then((res) => {
+        return res.rows;
+    })
```

Comment:
We only need one catch block at the end of our promise chain (in controller). Also a rejected promise doesn't then need to be thrown as an error. A rejected promise is already an error.